src/utils/dmm.ts

```diff
@@ -502,24 +503,7 @@ export function useRewardTokensFullInfo(): Token[] {
   const rewardTokens = useRewardTokens()
 
   const allTokens = useAllTokens()
-  const nativeName =
-    chainId && [137, 80001].includes(chainId)
-      ? 'MATIC'
-      : chainId && [97, 56].includes(chainId)
-      ? 'BNB'
-      : chainId && [43113, 43114].includes(chainId)
-      ? 'AVAX'
-      : chainId && [250].includes(chainId)
-      ? 'FTM'
-      : chainId && [25, 338].includes(chainId)
-      ? 'CRO'
-      : chainId && chainId === ChainId.BTTC
-      ? 'BTT'
-      : chainId && chainId === ChainId.VELAS
-      ? 'VLX'
-      : chainId && chainId === ChainId.OASIS
-      ? 'ROSE'
-      : 'ETH'
+  const nativeName = NETWORKS_INFO[chainId || ChainId.MAINNET].nativeToken.symbol
```

Comment:
I'm thinking about nativeOnChain function 
maybe that func and this config is duplicate, can we use nativeOnChain in network config?

Reply:
Yea it duplicate in someway. We may do a refactor for it later.